### ISSUE TITLE
Add `backgroundInfo` property to rules

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#1425]
There is often a lot of related information available for an issue, such as specs, related techniques, and for best practices there are related WCAG success criteria.

This solution was proposed in the ACT Rules Format, and has been requested for Assure. What we'd do is add a static `backgroundInfo` object to a rule, which should be included in the result:

```json
{
  "id": "foo",
  "backgroundInfo": {
    "https://www.w3.org/TR/WCAG21/#sign-language-prerecorded":
      "1.2.6 Sign Language (Prerecorded)",
    "https://www.w3.org/TR/WCAG20-TECHS/G54.html":
      "G54: Including a sign language interpreter in the video stream",
	"https://www.w3.org/TR/html-aria/#ahref":
      "Rules of ARIA attributes usage by HTML language feature"
  }
}
```

**Comment by dylanb** [comment]
This list of related information can change much more quickly than rules need to change, it should live outside axe-core. I suggest we close this issue

**Comment by WilcoFiers** [comment]
This kind of info might be better suited on DQU. Closing this issue.
